ivy/container/utility.py

```diff
@@ -10,44 +10,92 @@
 
 # noinspection PyMissingConstructor
 class ContainerWithUtility(ContainerBase):
+    @staticmethod
+    def static_all(
+        x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        axis: Optional[Union[int, Tuple[int], List[int], ivy.Container]] = None,
```

Comment:
we can replace `Tuple[int], List[int]` with `Sequence[int]` to be more concise :)

Reply:
I see! Will take note of this and make the changes! Thanks!